Tests/BreezeLambdaWebHookTests/BreezeLambdaWebHookTests.swift

```diff
@@ -13,79 +13,73 @@
 
-    let decoder = JSONDecoder()
-    let config = BreezeHTTPClientConfig(
-        timeout: .seconds(1),
-        logger: Logger(label: "test")
-    )
-    
-    init() {
-        setEnvironmentVar(name: "_HANDLER", value: "build/webhook", overwrite: true)
-        setEnvironmentVar(name: "LOCAL_LAMBDA_SERVER_ENABLED", value: "true", overwrite: true)
-    }
-    
-    deinit {
-        unsetenv("LOCAL_LAMBDA_SERVER_ENABLED")
-        unsetenv("_HANDLER")
-    }
+@Suite(.serialized)
+struct BreezeLambdaWebHookTests {
     
-    @Test("PostWhenMissingBody_ThenError")
-    func postWhenMissingBody_thenError() async throws {
-        let createRequest = try Fixtures.fixture(name: Fixtures.getWebHook, type: "json")
-        let request = try decoder.decode(APIGatewayV2Request.self, from: createRequest)
-        let apiResponse: APIGatewayV2Response = try await Lambda.test(MyPostWebHook.self, config: config, with: request)
-        let response: APIGatewayV2Response.BodyError = try apiResponse.decodeBody()
-        
-        #expect(apiResponse.statusCode == .badRequest)
-        #expect(apiResponse.headers == [ "Content-Type": "application/json" ])
-        #expect(response.error == "invalidRequest")
-    }
+    let decoder = JSONDecoder()
     
-    @Test("PostWhenBody_ThenValue")
-    func postWhenBody_thenValue() async throws {
-        let createRequest = try Fixtures.fixture(name: Fixtures.postWebHook, type: "json")
-        let request = try decoder.decode(APIGatewayV2Request.self, from: createRequest)
-        let apiResponse: APIGatewayV2Response = try await Lambda.test(MyPostWebHook.self, config: config, with: request)
-        let response: MyPostResponse = try apiResponse.decodeBody()
-        let body: MyPostRequest = try request.bodyObject()
-        
-        #expect(apiResponse.statusCode == .ok)
-        #expect(apiResponse.headers == [ "Content-Type": "application/json" ])
-        #expect(response.body == body.value)
-        #expect(response.handler == "build/webhook")
+    @Test("BreezeLambdaWebHook can be shutdown gracefully")
+    func breezeLambdaWebHookCanBeShutdownGracefully() async throws {
+        await testGracefulShutdown { gracefulShutdownTestTrigger in
+            let (gracefulStream, continuation) = AsyncStream<Void>.makeStream()
+            await withThrowingTaskGroup(of: Void.self) { group in
+                let logger = Logger(label: "test")
+                let config = BreezeHTTPClientConfig(timeout: .seconds(30), logger: logger)
+                let sut = BreezeLambdaWebHook<MockHandler>.init(name: "Test", config: config)
+                group.addTask {
+                    try await Task.sleep(nanoseconds: 1_000_000_000)
+                    gracefulShutdownTestTrigger.triggerGracefulShutdown()
+                }
+                group.addTask {
+                    await withGracefulShutdownHandler {
+                        do {
+                            try await sut.run()
```

Comment:
For some reason the ServiceGroup fails on the Lambda runtime and the gracefulShutdown is not being triggered.

```
◇ Test "BreezeLambdaWebHook can be shutdown gracefully" started.
2025-10-19T14:56:35+0000 info test: [BreezeLambdaWebHook] BreezeHTTPClientProvider: HTTPClient syncShutdown is completed.
2025-10-19T14:56:35+0000 info test: [BreezeLambdaWebHook] BreezeHTTPClientProvider: HTTPClient syncShutdown is completed.
2025-10-19T14:56:35+0000 info test: [BreezeLambdaWebHook] BreezeHTTPClientProvider: HTTPClient syncShutdown is completed.
2025-10-19T14:56:35+0000 info test: [BreezeLambdaWebHook] BreezeHTTPClientProvider: HTTPClient shutdown is completed.
2025-10-19T14:56:35+0000 info test: [BreezeLambdaWebHook] BreezeHTTPClientProvider started
```